images/win/scripts/Installers/Install-AndroidSDK.ps1

```diff
@@ -0,0 +1,147 @@
+################################################################################
+##  File:  Install-AndroidSDK.ps1
+##  Desc:  Install and update Android SDK and tools
+################################################################################
+
+# install command-line tools
+$cmdlineToolsUrl = "https://dl.google.com/android/repository/commandlinetools-win-7302050_latest.zip"
+$cmdlineToolsArchPath = Start-DownloadWithRetry -Url $cmdlineToolsUrl -Name "cmdline-tools.zip"
+$sdkInstallRoot = "C:\Program Files (x86)\Android\android-sdk"
+$sdkRoot = "C:\Android\android-sdk"
+Expand-Archive -Path $cmdlineToolsArchPath -DestinationPath "${sdkInstallRoot}\cmdline-tools" -Force
+# cmdline tools should be installed in ${sdkInstallRoot}\cmdline-tools\latest\bin, but archive contains ${sdkInstallRoot}\cmdline-tools\bin 
+# we need to create the proper folder structure
+Rename-Item "${sdkInstallRoot}\cmdline-tools\cmdline-tools" "latest"
+# ANDROID_NDK_PATH/HOME should not contain spaces. Otherwise, the script ndk-build.cmd gives an error https://github.com/actions/virtual-environments/issues/1122
+# create "C:\Android" directory and a hardlink inside pointed to sdk in Program Files
+New-Item -Path "C:\Android" -ItemType Directory
+New-Item -Path "$sdkRoot" -ItemType SymbolicLink -Value "$sdkInstallRoot"
+$sdkManager = "$sdkRoot\cmdline-tools\latest\bin\sdkmanager.bat"
+
+# Install the standard Android SDK licenses. Currently, there isn't a better way to do this,
+# so we are base64-encoded a zip of the licenses directory from another installation.
+# To create this base64 string, create a zip file that contains nothing but a 'licenses' folder,
+# which folder contains the accepted license files found in 'C:\Program Files (x86)\Android\android-sdk\licenses'.
+# Then, run this in PowerShell:
+#     $LicensesZipFileName = 'C:\Program Files (x86)\Android\android-sdk\Licenses.zip'
```

Comment:
Can it be done in this script instead of having that base64 string? it looks scary

Reply:
Nope, unfortunately this is the only workaround, which was in the previous iteration as well